code/nodes/runModels/utils.q

```diff
@@ -1,25 +1,30 @@
-\d .automl
+// code/nodes/runModels/utils.q - Utilities for the runModels node
+// Copyright (c) 2021 Kx Systems Inc
+//
+// Utility functions specific the the runModels node implementation
 
-// Utility functions for runmodels
+\d .automl
 
 // @kind function
 // @category runModelsUtility
-// @fileoverview Extraction of data from a file
-// @param filePath {str} File path from which to extract the data from 
-// @return {dict} parsed from file
+// @desc Extraction of data from a file
+// @param filePath {string} File path from which to extract the data from 
+// @return {dictionary} parsed from file
 runModels.i.readFile:{[filePath]
   key(!).("S=;")0:filePath
   }
 
 // @kind function
 // @category runModelsUtility
-// @fileoverview Fit and score custom model to holdout set
-// @param bestModel {sym} The best scorinng model from xval
-// @param tts       {dict} Feature and target data split into training and testing set
-// @param modelTab  {tab}  Models to be applied to feature data
+// @desc Fit and score custom model to holdout set
+// @param bestModel {symbol} The best scorinng model from xval
+// @param tts       {dictionary} Feature and target data split into training 
```

Comment:
Unnecessary spacing between param name and param type can be removed for the rest of the files